src/test/kotlin/com/bardsoftware/backend/fleet/rmarkdown/PandocTest.kt

```diff
@@ -63,11 +63,10 @@ class PandocTest {
                 .build()
                 .toByteString()
         val message = PubsubMessage.newBuilder().setData(request).build()
-        val isPublished = markdownReceiver.processMessage(message)
+        markdownReceiver.processMessage(message)
```

Comment:
Why don't you remove as well manipulations with that "result" flag and return type from `publish` function interface?

Reply:
you mean `onFailureCallback`?
Does it make any sense to use `isPublished` value then?
How do we want consume message in `CompilerFleetMessageReceiver`?

Reply:
We will need to refactor the decision of `ack`ing anyway. Basically, you want to ack when you started compilation process. In order to do this you'll need to run it asynchronously in `ExecutorService`. So, this is probably for the next pull requests.

Reply:
Done